superset/app.py

```diff
@@ -523,6 +523,9 @@ def init_views(self) -> None:
                     icon="fa-cog",
                 )
             appbuilder.add_separator("Data")
+        from superset.dashboards.filter_sets.api import FilterSetRestApi
+
+        appbuilder.add_api(FilterSetRestApi)
```

Comment:
```suggestion
        if feature_flag_manager.is_feature_enabled("DASHBOARD_NATIVE_FILTERS_SET"):
                from superset.dashboards.filter_sets.api import FilterSetRestApi
        
                appbuilder.add_api(FilterSetRestApi)
```
Lets add it under the feature flag